Windows: 'pthread.h': No such file or directory

```
C:\Users\syrel\Desktop\gtoolkit-vm\opensmalltalk-vm\ffiTestLibrary\src\callbacks.c(3,10): fatal error C1083: Cannot ope
n include file: 'pthread.h': No such file or directory [C:\Users\syrel\Desktop\gtoolkit-vm\build\build\ffiTestLibrary\T
estLibrary.vcxproj
```

A similar issue is with the `pThreadedFFI.h`
```
opensmalltalk-vm\ffi\include\pThreadedFFI.h(12,10): fatal error C1083: Cannot open include file: 'pthread.h': No such file or directory
```

--- a/ffi/include/pThreadedFFI.h
+++ b/ffi/include/pThreadedFFI.h
@@ -9,7 +9,11 @@
 #include <stdio.h>
 #include <ffi.h>
 #if FEATURE_THREADED_FFI
-#include <pthread.h>
+#ifdef _WIN32
+# include <Windows.h>
+#else
+# include <pthread.h>
+#endif
 #endif
 #include <errno.h>
 #include <stdlib.h>

--- a/ffiTestLibrary/src/callbacks.c
+++ b/ffiTestLibrary/src/callbacks.c
@@ -1,6 +1,10 @@
 #include "callbacks.h"
 #if FEATURE_THREADED_FFI
-#include <pthread.h>
+#ifdef _WIN32
+# include <Windows.h>
+#else
+# include <pthread.h>
+#endif
 #endif //FEATURE_THREADED_FFI
 
 EXPORT(int) singleCallToCallback(SIMPLE_CALLBACK fun, int base){
@@ -32,7 +36,13 @@ static int value = 0;
 #if FEATURE_THREADED_FFI
 void* otherThread(void* aFunction){
 	SIMPLE_CALLBACK f = (SIMPLE_CALLBACK) aFunction;
+	
+#ifdef _WIN32
+	Sleep(3);
+#else
 	sleep(3);
+#endif
+
 	value = f(42);
 }
 #endif //FEATURE_THREADED_FFI

